app/src/main/java/com/sdp/movemeet/backend/serialization/MessageSerializer.java

```diff
@@ -16,7 +16,7 @@
     // The key used to access the imageUrl attribute of a serialized Message
     public static final String MESSAGE_IMAGE_URL_KEY = "imageUrl";
 
-
+    @Override
```

Comment:
Thank you for pointing those "@Override" out to me! That's cool, I made the change on my side too!

Reply:
No problem ! Not sure they will make the coverage count for the interface, but we'll see :)